lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/base.scss

```diff
@@ -26,6 +26,14 @@ button, input, optgroup, select, textarea {
   line-height: inherit;
 }
 
+a {
+  color: var(--color-primary);
```

Comment:
I'd like to talk with @scriswell about what we want to do for default. Should we underline it?

Reply:
This was at @scriswell direction. The basic goal was to move away from purple links and he said to remove the underline.

Reply:
ok